author_api/models.py

```diff
@@ -30,9 +58,80 @@ def host(self):
     def url(self):
       return self.host + 'author/' + str(self.id)
 
+    objects = AuthorManager()
```

Comment:
Oooh, so basically, you're overriding the default Author.objects.blah() syntax so you can use the helper? Cool! Can you add that as a comment then? It's not immediately obvious.